config/routes.rb

```diff
@@ -111,6 +111,7 @@
       get "csv-download", to: "lettings_logs#download_csv"
       post "email-csv", to: "lettings_logs#email_csv"
       get "csv-confirmation", to: "lettings_logs#csv_confirmation"
+      get "update-logs", to: "lettings_logs#update_logs"
```

Comment:
Does this route anywhere? I don't see the corresponding action.

Reply:
It doesn't exist yet; I pulled this from Kat's PR to actually add the page. Without at least adding it to the routing, the URL construction for the E-mail fails. This seemed like the tidiest place to draw the line.